packages/solana/src/client.ts

```diff
@@ -330,9 +318,13 @@ export class Web3Modal extends Web3ModalScaffold {
     return SolStoreUtil.state.connection
   }
 
-  public async checkActiveProviders() {
+  public async checkActiveProviders(standardAdapter?: StandardWalletAdapter[]) {
```

Comment:
Maybe a plural name?
```suggestion
  public async checkActiveProviders(standardAdapters?: StandardWalletAdapter[]) {
```